tests/tensorflow/nightly/common.libsonnet

```diff
@@ -26,6 +27,20 @@ local mixins = import 'templates/mixins.libsonnet';
     },
     imageTag: 'nightly',
   },
+  // Setting the version for TPU VM.
+  tpuVm:: experimental.TensorFlowTpuVmMixin {
+    local config = self,
+    tpuSettings+: {
```

Comment:
In general, this directory should only have nightly tests. If these tests only work on 2.9, you can make the changes here instead: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/tensorflow/r2.9/keras-api.libsonnet